src/editorialPalette.ts

```diff
@@ -0,0 +1,102 @@
+// ----- Imports ----- //
+
+import {
+    text,
+    background,
+    neutral,
+    news,
+    opinion,
+    sport,
+    culture,
+    lifestyle,
+} from '@guardian/src-foundations/palette';
+import { Format, Design, Display, Pillar } from '@guardian/types/Format';
+
+
+// ----- Types ----- //
+
+interface Colour {
+    light: string;
+    dark: string;
+}
```

Comment:
This might be a bit restrictive for DCR. I don't have a specific example but my feeling is we often stray outside the bounds of 'neat'...

Reply:
I am assuming this is for dark mode?

Reply:
> I am assuming this is for dark mode?

Yep, many colours have a dark mode variant, the most prominent examples being the article background and body copy colour.